.github/workflows/js-api.yml

```diff
@@ -373,13 +375,29 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
         if: needs.version.outputs.changed_version == 'true'
+      - name: compatibility-tool
+        if: ${{ steps.publish.outcome == 'success' }}
+        working-directory: docusaurus
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
+          git pull
+          npm install
+          version=$(jq -r '.version' ../js-api/package.json)
+          echo "version - $version"
+          npx ts-node parser.ts $version          
+          git add ./static/versions
+          git commit -m "js-api version updated by Compatibility tool"
+          git sparse-checkout set static/versions
+          git push
```

Comment:
Check out Commit package-lock.json and its push mechanism, we use retried in case somebody committed smth before us